cmd/brew-portable-package.rb

```diff
@@ -1,28 +0,0 @@
-# Build a portable Homebrew utility bottle.
-#
-# Usage: brew portable-package <formula> [formula...]
-
-odie "Need to specify at least one portable formula!" if ARGV.empty?
-
-if RUBY_VERSION.split(".").first.to_i < 2
-  safe_system "brew", "install", "ruby"
-  ENV["HOMEBREW_RUBY_PATH"] = "#{HOMEBREW_PREFIX}/bin/ruby"
-end
-
-ENV["HOMEBREW_PREFER_64_BIT"] = "1"
-ENV["HOMEBREW_DEVELOPER"] = "1"
-
-ARGV.named.each do |name|
-  name = "portable-#{name}" unless name.start_with? "portable-"
```

Comment:
Why this is needed? At the same time, `test-bot` will only check formula in this tap. If we set up CI, test-bot will only test formula changed by PR.